discourse_migration/userguide.md

```diff
@@ -0,0 +1,141 @@
+| libc-commits          | no migration at the moment |
+| libc-dev              | lvm-project/libc |
+| Libclc-dev            | **TODO** |
+| libcxx-bugs           | no migration at the moment |
+| libcxx-commits        | no migration at the moment |
+| libcxx-dev            | lvm-project/libcxx |
+| lldb-commits          | no migration at the moment |
+| lldb-dev              | lvm-project/lldb |
+| llvm-admin            | **TODO** |
+| llvm-announce         | **TODO** |
+| llvm-branch-commits   | no migration at the moment |
+| llvm-bugs             | no migration at the moment |
+| llvm-commits          | no migration at the moment |
+| llvm-dev              | lvm-project/llvm |
+| llvm-devmeeting       | community/devmtg |
+| llvm-foundation       | **TODO** |
+| Mlir-commits          | no migration at the moment |
+| Openmp-commits        | no migration at the moment |
+| Openmp-dev            | lvm-project/openmp |
+| Parallel_libs-commits | no migration at the moment |
+| Parallel_libs-dev     | **TODO** |
+| Release-testers       | **TODO** |
+| Test-list             | **TODO** |
+| vmkit-commits         | **TODO** |
+| WiCT                  | **TODO** |
+| www-scripts           | **TODO** |
+
+**TODO:** add discourse links once the list is finalized
+
+## FAQ
+
+### I don't want to use a web UI
+
+You can do most of the communication with your email client (see section on
+Setting up email interactions above). You only need to set up your account once
+and then configure which categories you want to subscribe to.
+
+### How do I send a private message?
+
+On the mailing list you have the opportunity to reply only to the sender of
+the email, not to the entire list. That is not supported when replying via
+email on Discourse. However you can send someone a private message via the
+Web UI: Click on the user's name above a post and then on `Message`.
+
+Also Discourse does not expose users' email addresses , so your private
+replies have to go through their platform (unless you happen to know the
+email address of the user.)
+
+### How can my script/tool send automatic messages?**
+
+In case you want to [create a new
+post/topic](https://docs.discourse.org/#tag/Posts/paths/~1posts.json/post)
+automatically from a script or tool, you can use the
+[Discourse API](https://docs.discourse.org/).
+
+### Who are the admins for Discourse?
+
+See https://llvm.discourse.group/about
+
+### What is the reason for the migration?
+
+see https://lists.llvm.org/pipermail/llvm-dev/2021-June/150823.html
+
+### How do I set up a private mailing list?
+
+If needed categories can have individual [security
+settings](https://meta.discourse.org/t/how-to-use-category-security-settings-to-create-private-categories/87678)
+to limit visibility and write permissions. Contact the
+[admins(https://llvm.discourse.group/about) if you need such a category.
+
+### What will happen to our email archives?
+
+**TODO:** Explain the migration.
```

Comment:
Are we planning on redirecting existing URLs to the migrated Discourse threads?
There are messages in the archive that link to other message in the archive (see some of the links in this message for example: https://lists.llvm.org/pipermail/llvm-dev/2020-August/144012.html )

Reply:
I created a new issue for that: #46